server/src/test/java/org/apache/iotdb/db/integration/IoTDBMetadataFetchIT.java

```diff
@@ -180,11 +176,11 @@ public void databaseMetaDataTest() throws ClassNotFoundException, SQLException {
           .getConnection(Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root", "root");
       databaseMetaData = connection.getMetaData();
 
-      allColumns();
-      device();
-      showTimeseriesPath1();
-      showTimeseriesPath2();
-      showStorageGroup();
+//      allColumns();
+//      device();
+//      showTimeseriesPath1();
+//      showTimeseriesPath2();
+//      showStorageGroup();
```

Comment:
Please rewrite the tests.

Reply:
These tests are used to test `getColumn()` method which I have already disabled, `getColumn()` method is to be implemented in the future. I comment the code out because of maybe useful when  
`getColumn()` method is implemented.

Reply:
They are used to test the show statements instead of `getColumn()`, you just changed the way of using them and commenting them will make the functionality untested, which should be avoided.

Reply:
Show statement are already tested in `showStorageGroupTest`, `showTimeseriesTest()` of `IoTDBMetadataFetchIT`. `showTimeseriesPath1()`, `showTimeseriesPath1()` and `showStorageGroup()` are similar to above tests, the difference is using `getColumn()` method.
I think the only `show devices` needs to be tested.

Reply:
In that case, you should remove this code because `getColumn` is deprecated.

Reply:
fixed.